src/domain/customers/tableTools/customerMessageForm.module.scss

```diff
@@ -0,0 +1,33 @@
+@import 'spacings';
+
+.form {
+  > * {
+    margin: units(1) 0;
+  }
+  width: units(48);
+}
+
+.formActionButtons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.formActionButtonsRight {
+  display: flex;
+  justify-content: flex-end;
+
+  > button {
+    margin-left: units(0.5);
+  }
```

Comment:
and here :)

Reply:
same here :)

Reply:
We can give these buttons `className`s
https://github.com/City-of-Helsinki/berth-reservations-admin/blob/17ef7750ffcb4b5a04068670e119f9d649a57ed3/src/domain/individualHarbor/individualHarborTableTools/IndividualHarborTableTools.tsx#L20